### DISCUSSION TITLE
Does not work at all on macOS

### DISCUSSION THREAD

**Discussion by tangre** [xiaoxiaoflood/firefox-scripts#282] [General Support]
Followed all directions. Differences are having to extract the contents of fx-folder to [/Applications/Firefox.app/Contents/MacOS/firefox] and overwrite, and create "chrome" folder in [ /Users/XXXXX/Library/Application Support/Firefox/Profiles/zx5ptkd3.default-release] in which I extract "utils" creating a "utils" folder. I put the javascripts I wanted into the chrome folder, then did the clear cache and restart. Firefox does not seem to have reacted to this at all. When I attempt to install dta.xpi as instructed in the video, Firefox says it can't install because it "seems to be corrupted." So far as I can tell, this does not work anymore, or it doesn't work on macOS. I'm using Monterey, version 12.6.5, Firefox is 117.0.1 (64-bit).

**Comment by MoltenMonster** [comment]
@tangre How about you try it again and apply the changes here? https://github.com/xiaoxiaoflood/firefox-scripts/issues/265
